lib/puppet/provider/openldap.rb

```diff
@@ -15,7 +14,7 @@ def self.slapcat(filter, dn = '', base = 'cn=config')
       '-H', "ldap:///#{dn}???#{filter}"
     ]
 
-    if 'squeeze' == Facter.value(:lsbdistcodename)
+    if Facter.value(:lsbdistcodename) == 'squeeze'
```

Comment:
Squeeze is Debian 6 and that's no longer in metadata.json as supported. Should we drop it instead?

Reply:
:+1: